src/abt.sig

```diff
@@ -1,13 +1,37 @@
+(* This is the main point of interaction with this library. The ABT
+ * signature describes every operation generally needed to manipulate syntax.
+ *)
+
 signature ABT =
 sig
-  structure Symbol : SYMBOL
+  (* Any particular ABT is built upon several smaller concepts. Any of these
```

Comment:
How about "An implementation of ABTs is built on several smaller concepts, each of which may be tweaked."